src/js/modules/accounts/components/forms/SignInForm.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Field, reduxForm } from "redux-form";
+import injectSheet from "react-jss";
+import { EMAIL_REGEX } from "../../../../constants";
+
+const styles = {
+  errorMessage: {
+    color: "red",
+  },
+  successMessage: {
+    color: "green",
+  },
+};
+
+const validate = formValues => {
+  const errors = {};
+  if (!formValues.email) {
```

Comment:
Why not do this in the form above?

Reply:
Different forms may have different behaviors later on. I don't think it's necessary to pull it out, plus it may also clutter the files more.

Reply:
No, I mean the required vs invalid email address?